src/styles/stars.css

```diff
@@ -0,0 +1,19 @@
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  gap: 4px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: url("../images/star-active.svg") no-repeat center;
+  list-style: none;
+  gap: 4px;
+
+}
+
+.stars__star:last-child {
```

Comment:
you should manage the number of active stars using modifier (just copy all possible modifiers from the stars task)